src/sass/partials/base/_base.scss

```diff
@@ -0,0 +1,293 @@
+
+._wrapper {
+  @include wrapper;
+}
+
+.base {
+  &--a {
+    @include link(setting-get('active link color'), setting-get('active link color'), setting-get('secondary link color'), setting-get('secondary link color'));
+  }
+
+  &--h1 {
+    @include h1;
+  }
+
+  &--h2 {
+    @include h2;
+  }
+
+  &--h3 {
+    font-size: ms(2);
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h4 {
+    font-size: ms(1);
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h5 {
+    font-size: 1em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h6 {
+    font-size: ms(-1);
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--blockquote {
+    border-left: .25em solid setting-get('primary accent color');
+    color: color('gray', 80);
+    font-style: italic;
+    margin-left: 2em;
+    margin-right: 1em;
+    padding: 0 1.5em;
+  }
+
+  &--dt {
+    color: setting-get('secondary color');
+    font-weight: setting-get('bold font weight');
+  }
+
+  &--li {
+    margin-top: 0;
+  }
+
+  &--ul {
+    display: block;
+    list-style: circle;
+    margin-left: setting-get('rhythm');
+
+    @include breakpoint(setting-get('fixed container')) {
+      margin-left: 0;
+    }
+  }
+
+  &--ol {
+    margin-left: setting-get('rhythm');
+
+    @include breakpoint(setting-get('fixed container')) {
+      margin-left: 0;
+    }
+  }
+
+  &--hr {
+    border: 0 none;
+    border-top: 2px solid color('grey', 20);
+    margin: setting-get('rhythm') / .25;
+    margin-left: 0;
+    margin-right: 0;
+    width: 15%;
+  }
+
+  &--table {
+    margin-top: setting-get('rhythm') / .5;
+    width: 100%;
+  }
+
+  &--caption {
+    color: color('gray', 50);
+    font-size: .8em;
+    padding: setting-get('rhythm') / 2;
```

Comment:
Please use shorthand here instead of setting all of padding and then overriding it. `padding: top right bottom left` or `top horizontal bottom`.

For this, you should do `padding: 0 0 (setting-get('rhythm') / 2)`